File: src/Sandbox.Library.FSharp/Dtos/CustomerDto.fs
namespace Sandbox.Library.FSharp.Dtos

open System

type CustomerDto =
    struct
        val PartitionKey: string
        val SortKey: string
        val FirstName: string
        val LastName: string
        val Age: int
        val mutable UpdatedAt: DateTime
        val CreatedAt: DateTime
    end

File: src/Sandbox.Aws/CustomerDto.cs
namespace Sandbox.Aws
{
    using System;

    internal class CustomerDto
    {
        public string PartitionKey { get; set; }

        public string SortKey { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public int Age { get; set; }

        public DateTime UpdatedAt { get; set; }

        public DateTime CreatedAt { get; set; }
    }
}
